ee/get-support.md

```diff
@@ -45,6 +45,26 @@ This support dump only contains logs for the node where you're running the
 command. If your UCP is highly available, you should collect support dumps
 from all of the manager nodes.
 
+## Use the CLI to get a support dump (multiple nodes)
+
+To get the support dump from the CLI for multiple nodes, use SSH to log into a UCP manager node
+and run:
+
+To generate support dump details for all hosts in a cluster, from one remote host, please see below.
+
+The below variables need to be set initially and folders created prior to running the command. Please note the variables can be changed to fit your environment:
+
+```$ SUPPORT_DUMP_DIR=<^>/opt<^^>
+$ SUPPORT_DUMP_TEMPDIR=${SUPPORT_DUMP_DIR}/support_dump_temp
+$ DUMP_DATE=$(date +%Y%m%d-%H_%M_%S)
+$ mkdir ${SUPPORT_DUMP_TEMPDIR}
+$ UCP_VERSION=$(docker service inspect ucp-agent --format '{{range .Spec.TaskTemplate.ContainerSpec.Env}}{{$varval:= split . "="}}{{if eq "IMAGE_VERSION" (index $varval 0)}}{{index $varval 1}}{{end}}{{end}}')
```

Comment:
This is cool, but can you updated to use the version label instead? Use this:
```
UCP_VERSION=$(docker service inspect ucp-agent --format '{{index .Spec.TaskTemplate.ContainerSpec.Labels "com.docker.ucp.version"}}')
```

Reply:
completed.